packages/core/admin/ee/server/utils/features.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const { features } = require('@strapi/strapi/lib/utils/ee');
+
+const configFeatureFlags = {
+  'audit-logs': 'admin.auditLogs.enabled',
+  'review-workflows': 'admin.reviewWorkflows.enabled',
+};
+
+module.exports = Object.freeze({
+  ...features,
+  isEnabled(featureName) {
+    const featureEnabledInLicense = features.isEnabled(featureName);
+    const featureEnabledInConfig = configFeatureFlags[featureName]
+      ? strapi.config.get(configFeatureFlags[featureName], true)
+      : true;
+
+    return featureEnabledInLicense && featureEnabledInConfig;
+  },
+});
```

Comment:
That looks like something that should be inside `@strapi/strapi/lib/utils/ee`. But I also understand that we do not want to change the current behaviour of the `isEnabled` function inside it.
Could it be a topic for a refactoring initiative on EE service/utils?

Reply:
indeed, I would prefer not to put specific ee logic there. And yes I would say this should be a topic for the ee refactor 🚀 

I would say that all the license logic should actually be in the ee folder, not in core/strapi 🤔

Reply:
I think you were right, this piece of code should actually live in `@strapi/strapi/lib/utils/ee`, my thoughts:

- Frontend receives the list of activated features from `@strapi/strapi/lib/utils/ee` and it does not take into consideration the config feature flags.
- We already mention in `@strapi/strapi/lib/utils/license` which features apply to which license type, so It does not seem inapropiate to also mention which config flags exists per feature too.

I have added some changes, wdyt?